<?xml version="1.0"?>
<launch>



  <node name="vbacc" pkg="vbacc" type="vbacc_node" output="screen">
    <param name="cmdVelTopicName" value="/vrep/car/SetTwist" />
    <param name="cameraInfoName" value="/vrep/camera_info" />
    <param name="objectPolygonTopicName" value="/vision/predator_alert" />
    <param name="objectStatusPolygonTopicName" value="/vision/predator_alert/status" />

  </node>


</launch>
